client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/pattern-layout.tsx

```diff
@@ -24,125 +23,22 @@ type PatternLayoutProps = {
 	onDeleteFooter?: () => void;
 };
 
-// eslint-disable-next-line @typescript-eslint/no-empty-function
-const noop = () => {};
-
 const PatternLayout = ( {
-	header: selectedHeader,
 	sections,
-	footer: selectedFooter,
-	onAddHeader = noop,
-	onReplaceHeader = noop,
-	onDeleteHeader = noop,
 	onAddSection,
 	onReplaceSection,
 	onDeleteSection,
 	onMoveUpSection,
 	onMoveDownSection,
-	onAddFooter = noop,
-	onReplaceFooter = noop,
-	onDeleteFooter = noop,
 }: PatternLayoutProps ) => {
 	const translate = useTranslate();
 	const [ isPopoverVisible, setIsPopoverVisible ] = useState( false );
 	const addButtonContainerRef = useRef< HTMLDivElement | null >( null );
-	const isSidebarRevampEnabled = isEnabled( 'pattern-assembler/sidebar-revamp' );
-
-	if ( ! isSidebarRevampEnabled ) {
```

Comment:
Note: The feature flag is removed by https://github.com/Automattic/wp-calypso/pull/74149. We can ignore it from this PR